src/components/AvatarControl/AvatarControl.js

```diff
@@ -13,13 +14,20 @@ const AvatarControl = (props) => {
         <TextInput value={value}
                    onUpdate={(data) => onUpdate({value: data.value})} />
         <RandomButton onClick={ () => onUpdate({ value: AvatarControl.randomValue(props) }) }/>
+        {typeof value === 'undefined' ? 'undefined' : null}
+        {value === null ? 'null' : null}
       </label>
     </div>
   );
 };
 
-AvatarControl.randomValue = () => {
-  return faker.image.avatar();
+AvatarControl.randomValue = ({random = {}}) => {
+  const {
```

Comment:
I needed some way to attach a random function to a control. Since I don't want to fall back to extends Component I figured this would do. Do you have a better idea?